pkg/accounting/tally/tally.go

```diff
@@ -16,6 +17,7 @@ import (
 	"storj.io/storj/pkg/pb"
 	"storj.io/storj/pkg/pointerdb"
 	"storj.io/storj/pkg/storj"
+	satellitedb "storj.io/storj/satellite/satellitedb/dbx"
```

Comment:
Rest of the system shouldn't directly use dbx. Expose what you need via interfaces.